protostar/commands/test/testing_live_logger.py

```diff
@@ -81,9 +82,10 @@ def log(
                         if (
                             self.exit_first
                             and shared_tests_state.any_failed_or_broken()
+                            and not isinstance(test_result, NONNEGATIVE_RESULTS)
                         ):
                             tests_left_n = 0
-                            return
+                            continue
```

Comment:
Is this change correct? Shouldn't we break when we encounter a broken/failed test result?

Reply:
I believe we set test_left to 0 so in this case those would be equivalent? break may slightly better explain the intention but imo but this is a [[Nitpick]]